src/scdaq-gmt.conf

```diff
@@ -61,7 +61,7 @@ doZS:yes
 ##
 ################################################################################
 
-output_filename_prefix:scout_gmt
+output_filename_prefix:scout_GMT
```

Comment:
Same thing. In the "generic" rpm this should be commented -- it would be good to make noise if this is not set when you start the process but since we plan to send this via RCMS maybe it's not worth it ?

Reply:
commented out